### DISCUSSION TITLE
taro+vue3 引入element-plus

### DISCUSSION THREAD

**Discussion by wwm0806** [NervJS/taro#10893] [Q & A]
taro+vue3 引入element-plus报错问题，在webpack 中配置了
module: {
	rules: [
		{
			test: /\.mjs$/,
			include: /node_modules/,
			type: "javascript/auto"
		}
	]
}
依然步生效 请求大佬帮助

**Comment by maopixin** [comment]
taro一般都是写小程序的吧，element-plus不是后台组件库吗，要开发网页选taro实在不是明智之选
